apps/freelance/src/pages/JobDetailsPage/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Space } from 'antd';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { AvatarUpload, JobAppBar } from '@freelance/components';
+
+import {
+  JobDescrText,
+  JobOptionsText,
+  JobSkillsText,
+  LogoWrapper,
+  SkillsWrapper,
+  StyledButton,
+  Wrapper,
+} from './styles';
+
+export default function JobDetailsPage() {
+  const skills = ['skill1', 'skill2', 'skill3'];
```

Comment:
never use constants inside component, in this case initialization will be made with each rerender. move above the component

Reply:
fixed
![Screenshot 2022-11-23 105937](https://user-images.githubusercontent.com/87652792/203506502-f1b552ad-459e-4924-b66f-41125e3d75c4.jpg)